spec/features/fill_in_confirmation_of_supplied_details_page_spec.rb

```diff
@@ -8,21 +8,37 @@
   end
 
   scenario "correctly will enable user to continue to next page" do
-    given_i_am(:company01)
-    answer_all_questions
-
-    confirmation_of_supplied_details_page.submit_form
+    given_valid_data
+    start_a_new_et3_response
+    answer_respondents_details
+    answer_claimants_details
+    answer_earnings_and_benefits
+    answer_defend_claim_question
+    answer_representative
+    answer_disability_question
+    answer_employers_contract_claim
+    answer_additional_information
+    confirmation_of_supplied_details
 
     expect(form_submission_page).to be_displayed
   end
 
   scenario "incorrectly will provide errors" do
-    given_i_am(:erroneously_entering_data_on_confirmation_of_supplied_details_page)
+    @claimant = FactoryBot.create(:claimant, :claimant_valid)
```

Comment:
One for @Stivaros - if we can change the use of instance vars to using standard rspec let's

Reply:
Will not make this change following further discussion